.idea/copyright/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">

```

Comment:
this file shouldn't be in here
it is best to add `.idea` to your global htaccess